src/difference/difference_solution1/README.md

```diff
@@ -0,0 +1,83 @@
+<!-- BEGIN DOCS -->
+# Difference: finds the numbers that are in the first array but not in the second array
+<a name="arrayDifference"></a>
+
+## arrayDifference ⇒ <code>Array</code>
+
+This function creates an array of values that are in the first array,
+but not not in the second array.
+
+Repeated values are not duplicated in the return value,
+and the order of result values are determined by the first array.
+
+**Note:** This function returns a new array, and has no side-effects.
+
+**Returns**: <code>Array</code> - Returns the new array of filtered values.
+
+| Param                      | Type               | Default         | Description            |
+| -------------------------- | ------------------ | --------------- | ---------------------- |
+| [arrayToFindTheDifference] | <code>Array</code> | <code>[]</code> | The array to inspect.  |
+| [arrayToCompare]           | <code>Array</code> | <code>[]</code> | The values to exclude. |
+
+**Example**
+
+```js
+difference([2, 1], [2, 3]); //  [1]
+```
+
+**Example**
+
+```js
+difference([1, 2, 1], [2, 3]); //  [1]
+```
+
+> Docs generated: Wed May 19 2021, 1:46:52 AM
+
+
+## Strategy
+
+A additive approach is chosen to find the difference between two arrays. Starting from an empty array and added all the different elements to the array.
+
+---
+
+## Implementation
+
+Solution starts with an empty array . 
+
+Since the elements of first array which are not in the second array are  wanted, all the elements in the first array is searched one by one.
```

Comment:
In the implementation is missing language features and which lines of code can you use to make your strategy a reality?

Reply:
" implementation is missing language features and which lines of code can you use to make your strategy" If I do this as I understand it it will look like coding tutorial instead of a documentation

Reply:
Can you elaborate?